### ISSUE TITLE
After FFmpegFrameGrabber .close() ，The file is still occupied

### ISSUE THREAD

**Issue by JiGuangYuan** [bytedeco/javacv#1233]
Code like this

```

 FFmpegFrameGrabber grabber = new FFmpegFrameGrabber(new File("xxxx"));
        try {
            grabber.start();

        } catch (FrameGrabber.Exception e) {
            e.printStackTrace();
        }
.........
        try {
            grabber.close();
        } catch (FrameGrabber.Exception e) {
            e.printStackTrace();
        }
```
After FFmpegFrameGrabber .close() ，I can't delete this file.Prompt me that my file is occupied.
use JavaCV 1.5 ,Jdk 1.8

**Comment by saudet** [comment]
I cannot reproduce this issue here on Windows 10. This code delete the file just fine for me:
```java
File f = new File("video.mp4");
FFmpegFrameGrabber g = new FFmpegFrameGrabber(f);
g.start();
g.close();
f.delete();
```
This is probably caused by a faulty antivirus software. Please make sure to disable all malware scanners.

**Comment by JiGuangYuan** [comment]
I'm sorry.I double-checked the code, I found somewhere I invoke Twice start() ,then invoke close() ,The file is still occupied。

**Comment by saudet** [comment]
I see, we should add a check for that.

**Comment by saudet** [comment]
The check is now included in JavaCV 1.5.1. Thanks for reporting!
